### ISSUE TITLE
Why `loss.backward(loss)` in the Linear Regression example training

### ISSUE THREAD

**Issue by tmayer** [sotte/pytorch_tutorial#5]
Thanks for putting up the notebooks and the code! I enjoyed watching your presentation and following along with the notebooks. 

One question that bothered me is why you put the `loss` back in as the gradient for the `backward` method: `loss.backward(loss)` when training the linear regression network in the `01_pytorch_basics.ipynb` notebook. Is this just an error?

**Comment by sotte** [comment]
Ups, that should not be there! Sorry for the confusion and thanks a lot for finding that.

Fixed it.
